proto/osmosis/gamm/v1beta1/pool.proto

```diff
@@ -37,6 +37,20 @@ message PoolParams {
   ];
 }
 
+// PoolParamsWithoutLock is used to create the MsgCreatePool
+message PoolParamsWithoutLock {
```

Comment:
Why not just change the prior Pool Params to not have the lock?

Reply:
It could be used where chain/pool governance locks a pool through proposals if needed in the future. Is there a specific reason to need to remove this?

Reply:
Yeah, because the singular lock is insufficient, and actually there needs to be multiple locks.  See issue here:  https://github.com/osmosis-labs/osmosis/issues/68

So it makes sense to remove the Lock for now, and then add it back in, in the future when the locks can be done correctly (multiple types)

Reply:
I feel like permissioning is something that should eventually make it onto all 'hardcoded' AMMs